packages/components/psammead-most-read/src/__snapshots__/list.test.jsx.snap

```diff
@@ -93,6 +89,54 @@ exports[`MostReadList should render with ltr bengali items with correct dir 1`]
   }
 }
 
+@media (max-width:14.9375rem) {
```

Comment:
Where do these breakpoint values come from?

Reply:
The breakpoints are generated from the [parent grid breakpoints](https://github.com/bbc/psammead/blob/13136dae49840679e9440896f1d0f69e751aa872/packages/components/psammead-most-read/src/list.jsx#L49). Initially, [these](https://github.com/bbc/psammead/blob/13136dae49840679e9440896f1d0f69e751aa872/packages/components/psammead-most-read/src/list.jsx#L49) columns groups were not being applied to the child `Grids`, this PR seeks to fix that.